lib/floe/workflow/intrinsic_function/transformer.rb

```diff
@@ -151,6 +151,11 @@ def check_types(args, function, signature)
           OpenSSL::Digest.hexdigest(algorithm, data)
         end
 
+        rule(:states_json_to_string => {:args => subtree(:args)}) do
+          args = Transformer.process_args(args(), "States.JsonToString", [Object])
+          JSON.generate(args.first)
```

Comment:
I know it's pedantic, but for code consistency purposes
```suggestion
          json = args.first

          JSON.generate(json)
```